CMakeLists.txt

```diff
@@ -102,37 +102,30 @@ add_subdirectory(docs)
 add_dependencies(docs mlir-doc)
 
 # python install directory
-if (AIE_ENABLE_BINDINGS_PYTHON)
```

Comment:
I'm a little nervous about python being a hard dependency here.  MLIR practice has been for that to be optional, if only because getting it to build has lots of pitfalls.

Reply:
I guess one advantage of requiring python bindings is that aiecc.py could use them to walk the IR, rather than relying on aie-translate in a few places to extract information.